### PR TITLE
P4_16 v1.1.0 and P4_14 v1.0.5 release

### PR DESCRIPTION
**Pull request by jnfoster**
Final tweaks to change log, versions, and dates in preparation for release.

### CODE REVIEW

File: p4-16/spec/P4-16-spec.mdk [hunk 1]
```diff
@@ -6973,7 +6973,7 @@ The P4 compiler should provide:
 |-----|-----|-----|
 | 1.0.0 | May 17, 2017 | Initial version. |
 |-----|-----|-----|
-| 1.1.0 | June 5, 2018 | Added top-level functions, optional and named parameters, `enum` |
+| 1.1.0 | November 26, 2018 | Added top-level functions, optional and named parameters, `enum` |
 |       |          | representations, parser value sets, type definitions, saturating |
 |       |          | saturating arithmetic, and structured annotations. |
```
Comment (jafingerhut) [inline p4-16/spec/P4-16-spec.mdk]: "saturating" is repeated

Comment (jnfoster) [reply p4-16/spec/P4-16-spec.mdk]: You mentioned "rolling thunder" ... we are going for as many saturatings as possible :-)

**Comment by jnfoster** [comment]
@mbudiu-vmw I'll wait for your signal before merging and tagging these as the official release versions.

**Comment by mbudiu-vmw** [comment]
On a plane right now, will review tomorrow.

**Pull request merged**